### PR TITLE
4.3.0

### PR DESCRIPTION
**Pull request by Hoten**

### CODE REVIEW

File: changelog.md [hunk 1]
```diff
@@ -1,3 +1,88 @@
+ <a name="4.3.0"></a>
+# 4.3.0 (2019-04-08)
+[Full Changelog](https://github.com/googlechrome/lighthouse/compare/4.2.0...4.3.0)
+
+## New contributors!
+
+Thanks to @rdlabo and @yashrajbharti for their first contributions!
+
+## Clients
```
Comment (paulirish) [inline changelog.md]: i like the changelog groupings to be sorted by importance to the user.. something like: Report, Core, Docs, Clients, Tests, Misc, Deps

File: changelog.md [hunk 2]
```diff
@@ -1,3 +1,88 @@
...
+* add plugin recipe ([#6965](https://github.com/googlechrome/lighthouse/pull/6965))
+* readme: Foo integration ([#8035](https://github.com/googlechrome/lighthouse/pull/8035))
+* url-shim: add comment about public suffix lst ([#7671](https://github.com/googlechrome/lighthouse/pull/7671))
+
+## I18n
+
+* Hindi for seo and a11y ([#7786](https://github.com/googlechrome/lighthouse/pull/7786), [#8070](https://github.com/googlechrome/lighthouse/pull/8070))
+* add new translations for SEO; updates for a11y and perf ([#7720](https://github.com/googlechrome/lighthouse/pull/7720))
+
+## Renderer
```
Comment (paulirish) [inline changelog.md]: merge into report

File: changelog.md (hunk 1 above)
Comment (Hoten) [reply changelog.md]: done

File: changelog.md [hunk 3]
```diff
@@ -1,3 +1,85 @@
...
+* lr: add custom config support ([#7613](https://github.com/googlechrome/lighthouse/pull/7613))
+
+## Tests
+
+* better fatal trace logic error coverage ([#7959](https://github.com/googlechrome/lighthouse/pull/7959))
+* fix test coverage generation ([#7475](https://github.com/googlechrome/lighthouse/pull/7475))
+* finish removing mocha ([#7429](https://github.com/googlechrome/lighthouse/pull/7429))
+* upgrade jest to fix current node 11 build ([#7413](https://github.com/googlechrome/lighthouse/pull/7413))
+* smoke: remove trivial uses of rawValue from expectations ([#7780](https://github.com/googlechrome/lighthouse/pull/7780))
+* smoke: set Content-Type for files served by static-server.js ([#7612](https://github.com/googlechrome/lighthouse/pull/7612), [#8015](https://github.com/googlechrome/lighthouse/pull/8015), [#8078](https://github.com/googlechrome/lighthouse/pull/8078))
```
Comment (brendankenny) [inline changelog.md]: 👍

File: changelog.md (hunk 1 above)
Comment (patrickhulce) [reply changelog.md]: we should fix the script to make this automatic :)

**Pull request merged**